pkg/oc/cli/idle/idle.go

```diff
@@ -534,6 +535,33 @@ type scaleInfo struct {
 // scalable resources to zero, and annotating the associated endpoints objects with the scalable resources to unidle
 // when they receive traffic.
 func (o *IdleOptions) RunIdle() error {
+	networkPolicies, err := o.ClientSet.NetworkingV1().NetworkPolicies("").List(metav1.ListOptions{})
+
+	if err != nil {
+		return err
+	}
+
+	if len(networkPolicies.Items) > 0 {
+		for {
+			fmt.Fprint(o.ErrOut, "warning: idling when network policies are in place may cause connections to bypass network policy entirely. Continue (yes/no)? ")
+
+			var s string
+			_, err := fmt.Scan(&s)
+
+			if err != nil {
+				return err
+			}
+
+			s = strings.ToLower(s)
+
+			if s == "yes" || s == "y" {
+				break
+			} else if s == "no" || s == "n" {
```

Comment:
I think this should just be an else, otherwise any unexpected input would lead to idling.

Reply:
The ifs are enclosed within a for infinite loop.
Meaning, the exit points of the loop are either typing 'yes' or 'no' (and well, Ctrl-C)